Plugins/PackageToJS/Sources/PackageToJS.swift

```diff
@@ -8,6 +8,8 @@ struct PackageToJS {
         var configuration: String?
         /// Name of the package (default: lowercased Package.swift name)
         var packageName: String?
+        /// Target platform for the generated JavaScript (default: browser)
+        var platform: String?
```

Comment:
Could this become an `enum Platform: CaseIterable`, so then you would rely on `.allCases` to dynamically compute help string for all available platforms in the help output you've added below for this new option?